src/views/mypage/MyPageEditInfoView.vue

```diff
@@ -219,21 +233,6 @@ const handleCheckName = async () => {
   }
 };
 
-//현재 비밀번호 일치 여부 확인
-const validateCurrentPassword = () => {
-  if (!currentPassword.value.trim()) {
-    CurrnetPwdErrorMessage.value = '비밀번호를 입력해주세요.';
-    return false;
-  }
-  if (currentPassword.value !== member.password) {
-    CurrnetPwdErrorMessage.value = '비밀번호가 일치하지 않습니다.';
-    return false;
-  }
-  CurrnetPwdErrorMessage.value = '';
-  isCurrnetPwdChecked.value = true;
-  return true;
-};
-
 //새 비밀번호 일치 여부 확인
 const validateNewPassword = () => {
   if (!newPassword.value.trim() || !newPassword.value.trim()) {
```

Comment:
```suggestion
  if (!newPassword.value.trim() || !newPassword2.value.trim()) {
```
'newPassword2'인데 오타난 것 같아요!

Reply:
[6695ffd](https://github.com/kb-jjukkumi/choogoomoneyna_FE/pull/54/commits/6695ffd325a0fdca98d387678129a19e12ed41c6) 그러네요//!! 감사합니다 수정했습니당!